src/ml_data/column_statistics.cpp

```diff
@@ -237,11 +237,8 @@ void column_statistics::_finalize_global(
             // Adjust for many of the elements being zero
             sa.var_sum += std::pow(sa.mean, 2) * count * (1 - scale);
           }
-          if (total_row_count > 1) {
-            s.stdev = std::sqrt(sa.var_sum / (total_row_count - 1));
-          } else {
-            s.stdev = 0;
-          }
+
+          s.stdev = std::sqrt(sa.var_sum / (total_row_count - 1));
```

Comment:
Is this change intentional? It doesn't look related to Eigen.

Reply:
Good catch.